site/content/docs/5.2/helpers/focus-ring.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: Focus ring
+description: Utility classes that allows you to add and modify custom focus ring styles to elements and components.
+group: helpers
+toc: true
+---
+
+The `.focus-ring` helper removes the default `outline` on `:focus`, replacing it with a `box-shadow` that can be more broadly customized. The new shadow is made up of a series of CSS variables, inherited from the `:root` level, that can be modified for any element or component.
+
+## Example
+
+Click into the example below and press <kbd>Tab</kbd> to see the focus ring in action.
```

Comment:
Maybe we can tell that users can also simply click on the link to see the custom focus ring. IDK if it can be helpful but we could also remind folks that `:focus` is for keyboard and mouse in the description.